nodejs/awsx/apigateway/api.ts

```diff
@@ -399,7 +399,7 @@ function createLambdaPermissions(api: API, name: string, swaggerLambdas: Swagger
     const permissions: aws.lambda.Permission[] = [];
     for (const [path, lambdas] of swaggerLambdas) {
         for (const [method, lambda] of lambdas) {
-            const methodAndPath = `${method === "ANY" ? "*" : method}${path}`;
+            const methodAndPath = `${method === "ANY" ? "*" : method}/${path}`;
```

Comment:
this doesnt seem right.  the path will normally have the / in it already.  so this would likely break that, right?